app/components/channel_bookmarks/channel_bookmark/bookmark_icon.tsx

```diff
@@ -60,7 +60,7 @@ const BookmarkIcon = ({emoji, emojiSize, emojiStyle, file, genericStyle, iconSiz
             name='book-outline'
             size={22}
             color={theme.centerChannelColor}
-            style={genericStyle}
+            style={genericStyle as StyleProp<TextStyle>}
```

Comment:
In several files we are doing this, and the prop is only used to then be casted. Shouldn't we change the type directly in the prop?